solidity/contracts/converter/ConverterUpgrader.sol

```diff
@@ -215,4 +225,25 @@ contract BancorConverterUpgrader is IBancorConverterUpgrader, ContractRegistryCl
             }
         }
     }
+
+    bytes4 private constant IS_V28_OR_HIGHER_FUNC_SELECTOR = bytes4(keccak256("isV28OrHigher()"));
+
+    function isV28OrHigherConverter(IBancorConverter _converter) internal view returns (bool) {

```

Comment:
@yudilevi : Since we're not reusing this function from `BancorNetwork`, we may as well just implement and use function `converterType` in this file (similar to this function, but which does the whole work).

Reply:
But we'll need to know if the converter has a type or not anyway, no?
Otherwise, how will the converter know which type to create for older converters? Do it based on number of reserves?

Reply:
@yudilevi : Yes, but we may as well call `converter.converterType` (instead of `converter.isV28OrHigher`) via assembly.

Reply:
I'd rather have a single assembly function that we can also move somewhere (shared code) and then have specific logic in the clients.
Think about a case in which the version number was actually integer and accessible from all converters (including the old ones) - that's what I'm trying to emulate.

Reply:
@yudilevi : Of course, but the problem here is that we're NOT sharing it, we're duplicating it. So I think we may as well call it `getConverterType`, and inside it, do all the necessary logic (including some assembly) in order to get the converter's type.

Reply:
sharing it is the next step and is much preferable to writing multiple different functions.